aries-site/src/pages/whats-new.mdx

```diff
@@ -51,9 +51,17 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 --><!-- END: New Entry Template -->
 
 <!-- BEGIN: What's New Entries -->
+## July 26 - August 31, 2021 
+- Published the [Notification](https://design-system.hpe.design/components#visualizations) and [Toast Notifications](https://design-system.hpe.design/templates/toast-notifications) & [Status Indicator](https://design-system.hpe.design/templates/status-indicator) content with examples. 
+- Updated [toolbar button to include an icon only variant + changed stroke from 2px to 1px](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=0%3A1) 
+- Updated [Default Button](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=681%3A5164) in Figma library to align with Design System site by removing white background color in enabled state.
+- Updated name for "Tooltip" component in Figma library to ["Tip"](https://www.figma.com/file/Jd34cI5yyZIPb5BIwkIPU2/HPE-Tip?node-id=538%3A803) to align with the Design System site 
+- Added [numerical values to both sides of the RangeInput component](https://www.figma.com/file/BqCjvjc0rECQ4Ln2QhyjNi/HPE-Range-Input-Component?node-id=0%3A1) 
+- Cleaned up Ordering of Items Figma file. The actions [item is now the new toolbar button which was previously default button.](https://www.figma.com/file/TonE8uQ33e5QJRPknZx36q/Ordering-Of-Items) 
+
 ## July 19-23, 2021
 - Iterated on Notifications taxonomy based on research results.
-- Continued [ideating on written guidance for Navigation] (https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1%3A20).
+- Continued [ideating on written guidance for Navigation](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1%3A20).
```

Comment:
object of the sentence can be used as secondary choice of hyperlink as "continued ideating" (subject) doesn't have a great place to link to.

```suggestion
- Continued ideating on [written guidance for Navigation](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1%3A20).
```